### PR TITLE
Update chronicle version to 2024.09.0

### PR DESCRIPTION
**Pull request by matt-urbina**
### Background
Part of the Chronicle release described here: https://github.com/rstudio/chronicle/issues/1067. The Chronicle team has made a new code release and we're updating our helm to match.

### Summary
Updates Chronicle server values
Update our docs
Update the Connect/Workbench/PPM CI files to match these Chronicle updates
This requires bumping up the chart versions for Connect/Workbench/PPM as well

### CODE REVIEW

**Review by markrtucker — APPROVED** [review]
LGTM

**Comment by matt-urbina** [comment]
Hey @colearendt! When able, would we be able to get this PR Reviewed for our latest Chronicle Release? LMK if I need to tag anyone else!

**Review by kfeinauer — APPROVED** [review]

File: charts/rstudio-pm/NEWS.md [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
 
 ## 0.5.31
 
 - Add documentation about PostgreSQL database configuration and mounting passwords from secrets as an env variable
 
+
+## 0.5.31
+
+- Bump Chronicle Agent to version 2024.09.0
+
```
Comment (t-margheim) [inline charts/rstudio-pm/NEWS.md:14]: Should this still be here? Or is this now covered by the same note in `0.5.32`?

Comment (markrtucker) [reply charts/rstudio-pm/NEWS.md:14]: We should delete this

**Review by t-margheim — APPROVED** [review]
LGTM

**Review by markrtucker — APPROVED** [review]

**Pull request merged**